diagnostic.md

```diff
@@ -69,13 +66,13 @@ Using Atom, open up `diagnostic.md` and write your answers below (where indicate
 ```
 
 If we are in the `project` directory and use `cd planning`, is a relative or absolute path being referenced? How do you know?
-
- <!-- Answer Starts Here -->
+relative
```

Comment:
Correct, but it asks you to explain your answer: `planning` is referenced with respect to the current working directory, `project`.